src/components/dashboard/main-content/course-enrollments/CourseSection.jsx

```diff
@@ -28,6 +28,12 @@ class CourseSection extends React.Component {
     return <h2>{sectionTitle}</h2>;
   };
 
+  getFormattedOptionalSubtitle = () => {
+    const { subtitle } = this.props;
+    if (!subtitle) { return null; }
+    return <p className="mt-2">{subtitle}</p>;
```

Comment:
To clean up the spacing a bit more, might change the `className` for the subtitle to "mt-3 mb-0".